src/jobs/checkout_and_version.yml

```
@@ -27,9 +27,19 @@ executor: versioning
 steps:
   - checkout
 
-  - run:
-      name: Set version
-      command: /bin/git-version --dev-branch=<<parameters.dev_branch>> --release-branch=<<parameters.release_branch>> --version-prefix=<<parameters.version_prefix>> > .version
```

You just need to wrap the parameters in quotes:
```suggestion
      command: /bin/git-version --dev-branch="<<parameters.dev_branch>>" --release-branch="<<parameters.release_branch>>" --version-prefix="<<parameters.version_prefix>>" > .version
```

---

Nice suggestion 👌
Done ✅